scripts/entrypoint.sh

```diff
@@ -6,6 +6,8 @@ if [ -f tmp/pids/server.pid ]; then
   rm -f tmp/pids/server.pid
 fi
 
+export RELEASE_FOOTPRINT=$(date -u +'%Y-%m-%dT%H:%M:%SZ')
```

Comment:
You mentioned this will change every time the containers (re-)starts. Do you think it would make sense to also include a "stable" component, e.g. `git rev-parse --short HEAD`?

```sh
date=$(date -u +'%Y-%m-%dT%H:%M:%SZ')
sha=$(git rev-parse --short HEAD)
RELEASE_FOOTPRINT="${date} (${sha})"
```

Resulting in e.g. `2020-08-25T03:39:28Z (9a4873027)`.

Reply:
NVM, just read the original issue 🤦 It might still be interesting to see what is deployed, not when it was deployed, but that's clearly another topic. Sorry for the noise.